core/refinerycms-core.gemspec

```diff
@@ -2,7 +2,7 @@
 require File.expand_path('../../core/lib/refinery/version', __FILE__)
 
 version = Refinery::Version.to_s
-rails_version = ['>= 5.1.0', '< 5.2']
+rails_version = ['>= 5.2.0.rc1', '< 6']
```

Comment:
seems like this could stay at `>= 5.1.0` and you could specify the rc1 in the Gemfile

Reply:
At this time the only breaking change between Rails 5.1 and 5.2 is this : https://github.com/refinery/refinerycms/pull/3352/commits/70cd41cb09afec28ff65fc4b02dd636a9a201591 

So i'm ok to do what to say.

Any thoughts for this last remaining error `undefined method `except!' for nil:NilClass` ?

Reply:
Oh it looks like it's a rc1 bug, `mobility` gem has also this kind of problem: 
https://github.com/shioyama/mobility/issues/164
https://github.com/rails/rails/commit/5fcbdcfb574c731841be12764c50d9587b58345f

Reply:
seems like it might be addressed in the next rc